server/app.py

```diff
@@ -1,8 +1,32 @@
-from flask import Flask
+from flask import Flask, jsonify, request
+from flask_cors import CORS
+from datetime import datetime
 
 app = Flask(__name__)
-
+CORS(app)
```

Comment:
Security is a good thing, although it doesn't matter for now, we shouldn't allow all domain to access our server.

I think we should allow localhost only at the moment:
```python
allowed_origins = ["http://localhost:3000", "https://localhost:3000"]
CORS(app, origins=allowed_origins)
```